rector.php

```diff
@@ -37,5 +38,8 @@
     $rectorConfig->skip([
         CountOnNullRector::class,
         ExceptionHandlerTypehintRector::class,
+        UnionTypesRector::class => [
+            __DIR__.'/src/Controller/CRUDController.php',
```

Comment:
We maybe should add a comment like https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Util/TraversableToCollection.php#L27